planner/core/point_get_plan.go

```diff
@@ -334,36 +340,40 @@ func buildSchemaFromFields(ctx sessionctx.Context, dbName model.CIStr, tbl *mode
 	return schema
 }
 
-func getSingleTableName(tableRefs *ast.TableRefsClause) *ast.TableName {
+// return the ast node of queried table name and the alias string.
```

Comment:
Please follow the comment convention, for example: comment starting with function name / variable name; capitalize first character of beginning word and add `.` in the end of sentences.

Reply:
done